src/utils/generateEnvironmentVariable.js

```diff
@@ -0,0 +1,29 @@
+const VALUE = 'value'
```

Comment:
```suggestion
const ENV_VARIBALE_TYPE_VALUE = 'value'
```

and move it to global constants file